llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9817,10 +9833,10 @@ void LoopVectorizationPlanner::adjustRecipesForReductions(
       PhiR->setOperand(0, Plan->getOrAddLiveIn(RdxDesc.getSentinelValue()));
     }
   }
-
-  VPlanTransforms::clearReductionWrapFlags(*Plan);
   for (VPRecipeBase *R : ToDelete)
     R->eraseFromParent();
+
+  VPlanTransforms::runPass(VPlanTransforms::clearReductionWrapFlags, *Plan);
```

Comment:
I presume we needed to move `clearReductionWrapFlags` after the dead recipes were deleted to satisfy the verifier?

Reply:
Yep exactly